src/Symfony/Component/Form/composer.json

```diff
@@ -22,6 +22,7 @@
         "symfony/options-resolver": "~4.3|^5.0",
         "symfony/polyfill-ctype": "~1.8",
         "symfony/polyfill-mbstring": "~1.0",
+        "symfony/polyfill-php80": "^1.15",
```

Comment:
shouldnt be `1.16` according to https://github.com/symfony/polyfill-php80/commit/cb64c503a89a49bcf6334a57639e244e488b0fe9?

Reply:
it should, indeed

Reply:
good catch @noniagriconomie. fixed it